test/resources/router/partial/partial.js

```diff
@@ -25,6 +25,20 @@ r.get('/visible',
   }
 );
 
+r.get('/badvalidate',
```

Comment:
Maybe name the route `/undefinedValidateProp`?  I don't know if it technically is "bad" if the route can be created and tested.